src/OpenTK/Platform/Linux/LinuxNativeWindow.cs

```diff
@@ -374,6 +374,12 @@ public override Point PointToScreen(Point point)
             return new Point(point.X + client.X, point.Y + client.Y);
         }
 
+        public override bool CursorGrabbed
+        {
+            set {}
```

Comment:
Add a check that if "value == false" throw NotImplementedException. Ok to do nothing if value is true.